athena-vertica/src/main/java/com/amazonaws/connectors/athena/vertica/VerticaMetadataHandler.java

```diff
@@ -254,18 +255,24 @@ public void getPartitions(BlockWriter blockWriter, GetTableLayoutRequest request
         //get the bucket where export results wll be uploaded
         String s3ExportBucket = System.getenv(EXPORT_BUCKET_KEY);
         //build the SQL query
-
+        Random r = new Random();
+        int randomInt = r.nextInt(100) + 1;
+        String queryID = request.getQueryId().replace("-","").concat(String.valueOf(randomInt));
```

Comment:
Added code to use UUID.randomUUID().toString().
Needed this in case user issues multiple federated queries in a single query. Current design uses the Athena QueryId  as names for the S3 files generated by export. In case of multiple exports, the current design will cause conflicts.